integration/util/scenario/validator.js

```diff
@@ -166,9 +188,23 @@ class Validator {
     });
 
     const wsProvider = new WsProvider(`ws://localhost:${this.wsPort}`);
+    const types = await loadTypes(this.ctx);
     const api = await ApiPromise.create({
       provider: wsProvider,
-      types: await loadTypes(this.ctx)
+      types: {
+        ...types,
+        ApiAssetData: {
```

Comment:
dev note - not sure about this one .. just matching castle here

Reply:
removed.